addons/website_sale/static/src/scss/product_configurator.scss

```diff
@@ -65,11 +65,27 @@
         border-color: $input-border-color;
     }
 
+    .fa {
+        position: relative;
```

Comment:
Works fine for the position-relative and z-index but for the small the .fa class applies a font-size inherit which displays in 1rem because of the parent .btn class